lich.rbw

```diff
@@ -4920,10 +4922,22 @@ main_thread = Thread.new {
 
   @launch_data = nil
   require_relative("./lib/eaccess.rb")
-
+  
+  switch_index = ARGV.index("--entrydat")
+
+  if ( !switch_index.nil? )
+    # init.rb sees the unmodified ARGV breaking its logic
```

Comment:
Making GTK3 always/never required when using `--entrydat {file}` is counter to the intent, but it is also unclear why options like `--directory` and `--script-dir` would have any bearing on if the GUI is in use.

It /seems/ like that would be more tied to the `--login` and, maybe, `--without-frontend` options.

Reply:
Actually, it looks like the directory options aren't supported anymore but haven't been removed from the help output.  Options in the code:

In lich.rbw:
  --reconnect
  --login
  --gemstone
  --platinum
  --shattered
  --fallen
  --dragonrealms
  --test
  --genie
  --withoutfrontend

In init.rb:
  --wine-prefix=.+
  --wine=.+
  shellexecute
  --gui

Reply:
Ugh, I missed a bunch in lich.rbw:
  --start-scripts=(comma delimited list)
  --reconnect-delay=[0-9]+(?:\+[0-9]+)?
  --hosts-dir (directory)
  --detachable-client=[0-9]
  --stormfront (getopt short: -s)
  --avalon
  --wizard (getopt short: -w)
  --frostbite
  --game (getopt short: -g)

And maybe a bunch more starting around line 4561